index.html

```diff
@@ -0,0 +1,258 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"
+    />
+    <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+    crossorigin="anonymous"
+    referrerpolicy="no-referrer"
+  />
+
+    <link rel="stylesheet" href="style.css" />
+    <title>2023 Agricultural summit</title>
+  </head>
+  <body>
+    <header>
+    <ul class="head-links">
+        <li>
+          <a href="#"><i class="fab fa-facebook-f"></i></a>
+        </li>
+        <li>
+            <i class="fab fa-twitter"></i>
+        </li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#" class="login-btn">Logout</a></li>
+      </ul>
+    </header>
+    <nav>
+        <div class="ham"><i class="fa fa-bars"></i></div>
+       <div class="buger">
+        <p class="closeBtn"><i class="fa fa-times"></i></p>
+        <ul class="nav-js">
+            <li class="nav-jsi"><a href="index.html">Home</a></li>
+            <li class="nav-jsi"><a href="about.html">About</a></li>
+            <li class="nav-jsi">Program</li>
+            <li class="nav-jsi">Join</li>
+            <li class="nav-jsi">Sponsors</li>
+            <li class="nav-jsi news">News</li>
+        </ul>
+    </div>
+        <img src="images/LOGO.jpeg" alt="" class="main-logo">
+      <ul class="nav-ul">
+        <li class="nav-li"><a href="About.html">About</a></li>
+        <li class="nav-li">Program</li>
+        <li class="nav-li">Join</li>
+        <li class="nav-li">Sponsors</li>
+        <li class="nav-li news">News</li>
+      </ul>
+    </nav>
+    <section class="homepage">
+      <div>
+        <h3 class="greetings">"Hello World! No Agric, No life"</h3>
+        <h1 class="theme">
+          2023 AGRICUTURAL SUMMIT.THEME:SUSTAINNABLE LIVING:PRACTICAL SOLUTIONS
+          FOR A GREENER FUTURE
+        </h1>
+      </div>
+      <p class="paragraph">
+        A joyful celebration believing in the value of openness and sharing
+        creating a positive change with people from over 80 countries is taking
+        place @777 NE Martin Luther King Junior Blvd Portland, OR 97232 Oregon
+        U.S.A
+      </p>
+      <h2 class="date">2023.11.19(THU)-24(TUE)</h2>
+      <p class="venue">
+        @ 777 NE Martin Luther King Junior Blvd Portland,OR 97232 OREGON U.S.A
+      </p>
+    </section>
```

Comment:
- [ ] `required`
- [ ] Amazing job so far! 👏 
- [ ] However, I have a few suggestions according to the requirements.
- [ ] The pages should look almost identical to the original design. that includes, Colors.
- [ ] Typography: font face, size, and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
- [ ] Layout: composition and space between elements.

- [ ] please see screenshots below and see the highlighted areas and implement the suggestions

`your template`
![image](https://github.com/Danny-imo/Capstone-project/assets/66230306/75c101e1-2d16-43c8-ba55-6bc9e6799865)

`original`
![image](https://github.com/Danny-imo/Capstone-project/assets/66230306/77c78a8a-068a-47a5-9edf-852e760bdf3f)

Reply:
i want to contribute in this project